pytest.ini

```diff
@@ -1,2 +1,2 @@
 [pytest]
-addopts = -v -ra --cov
+addopts = -v -ra --cov --log-cli-level=DEBUG
```

Comment:
Just a heads-up that https://github.com/cisagov/skeleton-python-library/pull/139 is a thang now.